sdk/anomalydetector/azure-ai-anomalydetector/pom.xml

```diff
@@ -38,8 +38,8 @@
   </developers>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jacoco.min.linecoverage>0.2</jacoco.min.linecoverage>
```

Comment:
do you know why to downgrade the version?

Reply:
I'm not sure why the code coverage became less